src/parser/sparqlParser/SparqlQleverVisitor.cpp

```diff
@@ -1,20 +1,24 @@
 // Copyright 2021, University of Freiburg,
 // Chair of Algorithms and Data Structures
-// Author: Hannah Bast <bast@cs.uni-freiburg.de>
+// Authors:
+//   Hannah Bast <bast@cs.uni-freiburg.de>
+//   2022 Julian Mundhahs <mundhahj@tf.uni-freiburg.de>
 
-#include "SparqlQleverVisitor.h"
+#include <parser/sparqlParser/SparqlQleverVisitor.h>
```

Comment:
```suggestion
#include "parser/sparqlParser/SparqlQleverVisitor.h"
```
If I understand you correctly.

Reply:
This is not required to keep `clang-format` from reordering the includes. But the Google Style does it this way. And officially `#include <...>` seems to be for system headers. While `#include "..."` is for user defined headers with the fallback of system headers. So you're right. We should probably do it like you suggested.